src/tutorials/getting-started-with-drizzle-and-react.md

```diff
@@ -330,19 +277,26 @@ If the default Create-React-App page loaded without any issues, you may proceed.
 
```

Comment:
Also, should probably mention somewhere about MM needing permissions to connect to the app. (Something new they added since the tutorial).

Reply:
One last note pertaining to MM, after going through the tutorial, currently users still have to go into MM and set a custom network (http://127.0.0.1:9545) to connect to `truffle develop`. I notice that isn't mentioned here.

Also, once we merge the new `truffle develop` code, you can probably mention setting the port in the config for the develop network (maybe near where you mention configuring the `contracts_build_directory`?). Then users can just choose the `localhost 8545` option in the MM dropdown menu.

Reply:
I don't think MetaMask usage is within the scope of this tutorial, since the idea is to just beginners to have something running. Maybe something to add to the end of the article maybe? But I don't want to let that stop this PR from being merged, because it is broken right now.